CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Change Log
 
 ## [Unreleased]
+### Breaking
```

Comment:
NIT: Is this change breaking?
`:api-base-url` is optional and when `:api-base-url` is not specified, `send-url` and `bounce-url` are same as before.
Of course, `email*` , `retrieve-all-bounces*` and `delete-bounces*` functions are breaking, but these functions are private.
So it should be `Changed` in my opinion.

Reply:
`send-url` and `bounce-url` were public var before so I wrote as breaking.
But they were less likely used, I can agree with your opinion.

What do you think of?

Reply:
Oh, I overlooked `send-url` and `bounce-url` were public.
It is breaking indeed.
I reconsidered that `Breaking` is fine as it is.